src/js/tech/html5.js

```diff
@@ -1319,7 +1319,7 @@ const mp4RE = /^video\/mp4/i;
 
 Html5.patchCanPlayType = function() {
   // Android 4.0 and above can play HLS to some extent but it reports being unable to do so
-  if (browser.ANDROID_VERSION >= 4.0) {
+  if (browser.ANDROID_VERSION >= 4.0 && !browser.IS_FIREFOX) {
```

Comment:
I just saw an update to the firefox beta which said they're getting HLS support. Is this something that we should possibly account for? Maybe check if firefox version is less than 50?